Src/LinearSolvers/MLMG/AMReX_MLMG.cpp

```diff
@@ -1113,10 +1126,9 @@ MLMG::prepareForSolve (const Vector<MultiFab*>& a_sol, const Vector<MultiFab con
     sol_raii.resize(namrlevs);
     for (int alev = 0; alev < namrlevs; ++alev)
     {
-        if (a_sol[alev]->nGrow() == 1)
+        if (a_sol[alev]->nGrow() >= 1)
         {
             sol[alev] = a_sol[alev];
-            sol[alev]->setBndry(0.0);
```

Comment:
We probably rely on this in some parts of the code.  So we need to put this back.

Reply:
Done (see previous comment)